seq2science/rules/configuration_explain.smk

```diff
@@ -108,7 +108,7 @@ else:
             assert isinstance(ret,dict), f"Invalid config key '{opt}' in {opts}."
             ret = ret.get(opt)
             if ret in [None, ""]:
-                return ""
+                return " with default options"
```

Comment:
@siebrenf would this work, or are there edge cases I don't think of? (and the tests miss those)